CPS-????/README.md

```diff
@@ -0,0 +1,93 @@
+---
+CPS: ?
+Title: Coin Selection Approach for Post Native Tokens Era
```

Comment:
```suggestion
Title: Coin Selection Including Native Tokens
```
Changing to proposed title from https://github.com/cardano-foundation/CIPs/pull/609#discussion_r1367769300 because we need the fewest number of the most significant words possible that uniquely characterise each CIP.  Open to further changes if anyone thinks the "Era" is significant to understanding & solving this problem.